### PR TITLE
Issue 592 ttlmon rtstat

### PR DESCRIPTION
**Pull request by mortenjc**
### Issue reference / description

The branch you merge from should already reference an event-formation-unit github ticket number. You can add a descriptive title, but if an issue is referenced, you don't have to.

## Checklist for submitter

- [ ] Check for conflict with integration test
- [ ] Unit tests pass

---

### Nominate for Group Code Review

- [ ] Nominate for code review

### CODE REVIEW

File: src/modules/ttlmonitor/test/TTLMonitorInstrumentTest.cpp [hunk 1]
```diff
@@ -161,10 +159,8 @@ TEST_F(TTLMonitorInstrumentTest, BeamMonitorTOF) {
   ttlmonitor->ESSReadoutParser.Packet.Time.setReference(1, 100000);
   ttlmonitor->ESSReadoutParser.Packet.Time.setPrevReference(1, 0);
 
-  auto Readouts =
-      ttlmonitor->TTLMonParser.parse(ttlmonitor->ESSReadoutParser.Packet);
+  ttlmonitor->TTLMonParser.parse(ttlmonitor->ESSReadoutParser.Packet);
       counters.TTLMonStats = ttlmonitor->TTLMonParser.Stats;
-  ASSERT_EQ(Readouts, 1);
```
Comment (JennyLouise) [inline src/modules/ttlmonitor/test/TTLMonitorInstrumentTest.cpp:167]: Did asserting the result == 0 mean checking no errors had occurred? Can we replace this check in the tests with error counter checks?

File: src/modules/ttlmonitor/test/ParserTest.cpp [hunk 2]
```diff
@@ -106,8 +113,7 @@ TEST_F(TTLMonitorParserTest, ErrorHdrDataSize) {
 TEST_F(TTLMonitorParserTest, ErrorDataSize) {
   makeHeader(DataBadSize);
 
-  auto Res = parser.parse(PacketData);
-  ASSERT_EQ(Res, 0);
+  parser.parse(PacketData);
```
Comment (JennyLouise) [inline src/modules/ttlmonitor/test/ParserTest.cpp:116]: Can the ASSERT_EQ(Res, 0) line be replaced with some error counter checks? Did res == 0 mean that no errors occurred while parsing?

Comment (mortenjc) [reply src/modules/ttlmonitor/test/ParserTest.cpp:116]: Yes, it really is redundant and was only used in tests. It returned the number of good readouts, but that is also covered in the stat counters and in the length ofthe array containing the list of successfully parsed readouts.

Comment (JennyLouise) [reply src/modules/ttlmonitor/test/ParserTest.cpp:116]: Okay, as long as those counters are being checked than I'm happy with this change

**Review by JennyLouise — APPROVED** [review]

**Pull request merged**